content/c/concepts/basic-output/basic-output

```diff
@@ -0,0 +1,71 @@
+---
+Title: "Basic Output"
+Subjects:
+  - "Code Foundations"
+  - "Computer Science"
+Tags:
+  - "Print"
+  - "Output"
```

Comment:
We still need a `CatalogContent` even though it's not C content:

```
CatalogContent:
  - "learn-c-plus-plus"
  - "paths/computer-science"
```

Reply:
Done. Is this something that could be changed in the future, when there isn't any content close to it?

Reply:
The only reason we have this right now is to have course recommendations just in case the reader wants to learn more. So the dream is, we just make a Learn C course. Hopefully, we can use the data from Codecademy Docs to show people that we should have a Learn C course. I know it's been talked about throughout the years, but never been made.